### ISSUE TITLE
Create a project website

### ISSUE THREAD

**Issue by lresende** [elyra-ai/elyra#92]
The project would benefit from a better presence on the web where high-level information related to project capabilities and components could be provided for users. 

Things we should have in mind:
- Elyra open source branding and messaging
- Elyra project vs Elyra subcomponents (e.g. Canvas)
- Theme Consistency
- Ability to land on the project page or sub-project page
- Others?

**Comment by mrmath3w** [comment]
the concept and design looks awesome! it'll be great to see this live as a landing experience for Elyra

**Comment by lresende** [comment]
@quarryboy This looks great, please let me know when you have something that can come as a pr and we can work together on the infra side. As for the Elyra "Notebook" content, I will help with moving some of our existing content to this format.

@ptitzler Let me know your thoughts on this area.

**Comment by quarryboy** [comment]
@lresende will do. i'll work on getting the content sorted out over the coming weeks, review that content here, and then i can produce a PR. looking forward to other recommendations as they may appear here.

**Comment by ajbozarth** [comment]
The design looks good, it seems a bit dark though. Would it work to switch the black out for something off-black so we can still get that l&f but lighten it top just a touch?

**Comment by quarryboy** [comment]
@ajbozarth we could definitely do that with the homescreen but i believe the aesthetic of the getting started, FAQs, components, etc. are all set in the theme in terms of darkness.

**Comment by ptitzler** [comment]
For _Elyra (sans canvas)_ I'd envision a landing page that provides access to the following resources (listed below in no particular order):
 - a very brief high level overview (a much shorter version of https://elyra.readthedocs.io/en/stable/getting_started/overview.html)
 - the documentation (https://elyra.readthedocs.io/en/stable/)
 - the github repository (https://github.com/elyra-ai/elyra)
 - community resources (essentially the content from https://elyra.readthedocs.io/en/stable/getting_started/getting-help.html)
 - announcements/what's new (in essence links to third party publications with Elyra content, such as blog posts)

The main motivation is that we currently don't have a single entry point on the web that provides access to all of the above. 

I would try to avoid duplicating documentation content  and aim for content that is mostly static and doesn't have to undergo frequent changes.

**Comment by quarryboy** [comment]
oh i definitely want to steal the announcements for canvas as well. thanks for the idea, @ptitzler !

**Comment by lresende** [comment]
@ptitzler what would you envision the division of contents in the context of the website? Do we consider the support for JupyterLab the main content (which is what we have today on readthedocs) or do we start partitioning it, so we can have, let's say, a section for VSCode, and other places we might support?

**Comment by ptitzler** [comment]
TL;DR: probably separate topics where required

Assuming that the current readthedocs content remains on readthedocs _for now_ and is just linked to from the new webpage it'd probably favor one main feature overview (pipelines[editor], Python editor, ...) with each feature section linking to the specific RTD section for each supported IDE. For example (in no particular order):
- Python editor overview -> links only to RTD/JL-topic (assuming there's no Elyra Python editor support in VSC)
- Pipeline [editor] overview -> links to RTD/JL topic _and_ links to RTD/VSC topic
- CLI -> links to common RTD topic
- ...
